src/new-block/components/gravatar-card-icon.tsx

```diff
@@ -0,0 +1,18 @@
+export default function GravatarCardIcon() {
+	return (
+		<svg
```

Comment:
Gutenberg uses a `SVG` and `Path` component which we can probably use here:

https://github.com/WordPress/gutenberg/blob/3fd7c8a2cbdb321611c447d64d6fe4b79dfdefd0/packages/primitives/src/svg/README.md?plain=1

Reply:
✅